GDJS/Runtime/InAppTutorialMessage.tsx

```diff
@@ -108,7 +109,7 @@ namespace gdjs {
         .then((fontFace) => document.fonts.add(fontFace));
     };
 
-    const _loadStyles = () => {
+    const _loadStyleSheet = () => {
       const adhocStyle = document.createElement('style');
```

Comment:
Should you store a reference to it to avoid re-creating it each time _loadStylesheet is called?